ui/cypress/integration/admin.test.ts

```diff
@@ -0,0 +1,259 @@
+describe('Use Admin tab', () => {
+  let url: string
+  let sourceId: string
+
+  beforeEach(() => {
+    cy.OAuthLogin('test').then(() => {
+      cy.visit('/login')
+      cy.removeConnections()
+      cy.createConnection()
+      cy.get('@connections').then(source => {
+        sourceId = source[0].id
+      })
+    })
+  })
+
+  //   describe('Chronograf', () => {
+  //     beforeEach(() => {
+  //       cy.fixture('routes').then(({adminChronograf}) => {
+  //         url = `/sources/${sourceId}${adminChronograf}`
+  //       })
+  //     })
+
+  //     /* ADMIN TAB CHRONOGRAF */
+  //   })
+
+  describe.only('InfluxDB', () => {
+    beforeEach(() => {
+      cy.fixture('routes').then(({adminInfluxDB}) => {
+        url = `/sources/${sourceId}${adminInfluxDB}`
```

Comment:
Application URI space (navigation paths, HTTP API paths) is a part of a design choice, it is not an implementation detail that can easily change. The paths are thus not dereferenced even in the application code. There is IMHO no need to resolve the paths using fixtures. Just FYI

Reply:
Fixture file removed and tests adjusted.